src/services/lock/service/lock.service.ts

```diff
@@ -150,16 +151,16 @@ export class LockService {
     const dist1 = distance(Buffer.from('foo'), Buffer.from('bar'));
     const nodes = await this.lockNodeRepository.distinct('node_id');
     const nodesClosest = nodes
-      .map((e) => {
+      .map<[number, string]>((e) => {
```

Comment:
Can't this be inferred too? If it can't, at least make it the function return type instead.
```suggestion
      .map((e) => {
```
```suggestion
      .map((e): [number, string] => {
```